### DISCUSSION TITLE
pngquant pre-build test failed in MacOS

### DISCUSSION THREAD

**Discussion by m1911star** [ueberdosis/tiptap#1247] [Showcase]
If you exec `yarn`, and see errorbelow, please install `libpng` and `pkg-config` with brew, and everything is ok 

![image](https://user-images.githubusercontent.com/4948120/116672285-97362100-a9d4-11eb-82d0-c33b84d301e0.png)

```bash
brew install libpng
brew install pkg-config
```

**Comment by hanspagel** [comment]
I’m not sure this is related to tiptap at all.

**Reply by m1911star** [reply]
Sorry. Just run into an error while npm installing. If not fit, I will delete
